{"uuid": "a844701d-94c4-4496-a215-7af36198c009", "children": ["5d725791-8676-4978-ace6-757ee1df2d8c"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741860430368, "stop": 1741860430369}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 112, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741860430583, "stop": 1741860430586}], "start": 1741860430368, "stop": 1741860430586}